src/status_im2/contexts/communities/discover/events.cljs

```diff
@@ -37,6 +37,7 @@
                  :other    (select-keys (:communities cc) other)})}))
 
 (rf/defn fetch-contract-communities
+  {:events [:fetch-contract-communities]}
```

Comment:
curious, how was this even called before without an event name? 🤔

Reply:
It wasn't, but communities were retrieved every 10s I believe (seems to often to be honest), but I think it still best for UX to retrieve them on landing on the page, just in case